### ISSUE TITLE
MISP Circl Feed - Events published but no notifications

### ISSUE THREAD

**Issue by yokoulaaa** [MISP/MISP#7198]
We have enabled CIRCL default feed on our MISP Instance. I see no auto-publish option given the format (MISP) compared to csv. 
However, I see the events in "Published" state on our local instance when I Fetch and pull them manually. But we receive no email notifications. We don't have this problem for events published by users. 
It looks like the events are not really published, because our TheHive instance doesn't pull these CIRCL events from MISP, but can pull those from users. 

Moreover, events from CIRCL are not automatically pulled, I have to "Fetch and pull all events" manually. 

*If you would like to report a bug, please fill the template bellow*

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)   | CentOS 7
| OS version (client)     | CentOS 8
| PHP version               | 7.2.24
| MISP version / git hash   | 2.4.136 (9edf4c189083576eed2a6c082bc5b7d5052193d9) 

### Expected behavior
I'd like CIRCL feed to publish events on our local instance without doing it manually. And I'd like to receive notifications about them when they are published.

### Actual behavior
No auto publish and we receive no notifications by emails. No problem with events published by users.

**Comment by chrisinmtown** [comment]
To fetch data from default feeds automatically, add a crontab entry that invokes the Cake/MISP feature, supplying as IDs numeric identifiers from the users and feeds tables, respectively:

    /var/www/MISP/app/Console/cake Server fetchFeed ${userId} ${feedId}

As for email notifications, a daily pull of default feeds could update 50+ events with 1000+ new attributes. Are you asking for an email for every attribute added? Every event unpublished/republished?

**Comment by yokoulaaa** [comment]
Thank you so much for the response ! I'll try this 

For email notifications, I was asking for every event published by feeds (CIRCL)

**Comment by yokoulaaa** [comment]
When I fetch data from CIRCL feed manually, I see events push to my instance but there aren't any notification even though they have a "Published" status. 

![image](https://user-images.githubusercontent.com/65095610/110946213-e77ef400-833e-11eb-9e13-0d48f13cef79.png)

**Comment by adulau** [comment]
The email notification depends of the configuration of your MISP instance. Did you have the email notification enabled? Are the SMTP parameters correct on your server?

**Comment by yokoulaaa** [comment]
Yes, we have enabled it and it works for events created and published by users on our instance.

**Comment by gacyn** [comment]
You may try to turn off ```MISP.disablerestalert``` and beware of email flooding.
